typescript/raven.d.ts

```diff
@@ -1,287 +1,301 @@
+     * executed immediately.
      *
-     * @param {object} options A specific set of options for this context [optional]
-     * @param {function} func The callback to be immediately executed within the context
-     * @param {array} args An array of arguments to be called with the callback [optional]
+     * @param options A specific set of options for this context
+     * @param func The callback to be immediately executed within the context
+     * @param args An array of arguments to be called with the callback
      */
     context(func: Function, ...args: any[]): void;
     context(options: RavenWrapOptions, func: Function, ...args: any[]): void;
 
-    /*
+    /**
      * Wrap code within a context and returns back a new function to be executed
      *
-     * @param {object} options A specific set of options for this context [optional]
-     * @param {function} func The function to be wrapped in a new context
-     * @return {function} The newly wrapped functions with a context
+     * @param options A specific set of options for this context
+     * @param func The function to be wrapped in a new context
+     * @return The newly wrapped functions with a context
      */
     wrap(func: Function): Function;
     wrap(options: RavenWrapOptions, func: Function): Function;
     wrap<T extends Function>(func: T): T;
     wrap<T extends Function>(options: RavenWrapOptions, func: T): T;
 
-    /*
+    /**
      * Uninstalls the global error handler.
-     *
-     * @return {Raven}
      */
     uninstall(): RavenStatic;
 
-    /*
+    /**
      * Manually capture an exception and send it over to Sentry
      *
-     * @param {error} ex An exception to be logged
-     * @param {object} options A specific set of options for this error [optional]
-     * @return {Raven}
+     * @param ex An exception to be logged
+     * @param options A specific set of options for this error
      */
     captureException(ex: Error, options?: RavenOptions): RavenStatic;
 
     /*
      * Manually send a message to Sentry
      *
-     * @param {string} msg A plain message to be captured in Sentry
-     * @param {object} options A specific set of options for this message [optional]
-     * @return {Raven}
+     * @param msg A plain message to be captured in Sentry
+     * @param options A specific set of options for this message
      */
     captureMessage(msg: string, options?: RavenOptions): RavenStatic;
 
-    /** Log a breadcrumb */
-    captureBreadcrumb(crumb: Object): RavenStatic;
+    /** 
+     * Add a breadcrumb
+     * @param crumb The trail which should be added to the trail
+     */
+    captureBreadcrumb(crumb: RavenBreadcrumb): RavenStatic;
+
+    /**
+     * Set a user to be sent along with payloads.
+     *
+     * @param user The definition of the currently active user's unique identity
+     */
+    setUserContext(user: RavenUserContext): RavenStatic;
```

Comment:
But if the overload is removed, the difference in the documentation is lost.

As the function behaves (a bit) differently, if you pass it nothing or a `RavenUserContext`, I wanted to also reflect that in the inline-/tooltip -documentation